store/postgres/secret_repository.go

```diff
@@ -86,56 +85,81 @@ func (p Secret) ToSpec(hash models.ApplicationKey) (models.ProjectSecretItem, er
 	}, nil
 }
 
+func (p Secret) ToSecretItemInfo(hash models.ApplicationKey) (models.SecretItemInfo, error) {
```

Comment:
doesn't use hash anymore can be removed, can't we name it as key instead of hash in other places.
We will move the logic of encryption & decryption to service layer can you add that as todo

Reply:
Will remove the param, and rename in other places. Sure will add todo